modules/enrichmentFpdModule.js

```diff
@@ -37,7 +37,10 @@ function setDomain() {
 
   let domain = parseDomain(getRefererInfo().canonicalUrl)
 
-  if (domain) utils.mergeDeep(ortb2, { site: { domain: domain } });
+  if (domain) {
+    utils.mergeDeep(ortb2, { site: { domain: domain } });
+    utils.mergeDeep(ortb2, { site: { publisher: { domain: domain } } });
```

Comment:
Prebid Server defines these values differently. site.domain is meant to be the 'raw' domain. e.g. www.example.com. site.publisher.domain is the 'aggregated' version of this domain -- just example.com in this instance.

So I'd propose we keep this convention in Prebid.js as well. i.e. implement/share the findRootDomain  function in modules/userId/index.js

If we don't, then site.publisher.domain will be passed raw to PBS and won't be  aggregated there.

Reply:
@bretg, thank you for this clarification. I just need to know one more thing, does the root domain include subdomains as well?

for example, if the root domain is `mydomain.example.com`, then would we use `example.com` or `mydomain.example.com` as the value for site.publisher.domain.

Reply:
the 'root' domain is meant to be the broadest possible domain you can still set a cookie in. So in your example, it would be `example.com`. As another example, the root domain for www.sports.example.co.uk is example.co.uk